scripts/speech_llm/export_conversations_to_tar.py

```diff
@@ -0,0 +1,28 @@
+from random import Random
+
+import click
+from lhotse import CutSet
+
+from nemo.collections.common.data.lhotse.text_adapters import (
+    NeMoMultimodalConversationJsonlAdapter,
+    NeMoMultimodalConversationTarWriter,
+)
+
+
+@click.command()
+@click.argument("manifest", type=click.Path())
+@click.argument("output_dir", type=click.Path())
+@click.option("-n", "--shard_size", type=int, default=100, help="Number of conversations per shard.")
+@click.option("--shuffle/--no-shuffle", default=False, help="Shuffle conversations.")
+@click.option("-s", "--seed", type=int, default=42, help="Random seed.")
+def export(manifest: str, output_dir: str, shard_size: int, shuffle: bool, seed: int):
+    with NeMoMultimodalConversationTarWriter(output_dir, shard_size=shard_size) as writer:
```

Comment:
will this tarring support out of the box on speech-to-speech case where there is speech in the assistant turns? if not, can u document what may be missing?

Reply:
Yes, it will support this out of the box.